modules/backup-and-restore/pages/database-import-export.adoc

```diff
@@ -159,8 +159,9 @@ The security features prevent users from directly unzipping it.
 For each vertex or edge type called `<type>`, there is one of the following two data files:
 ** `vertex_<type>.csv`
 ** `edge_<type>.csv`
-* `global.gsql` - DDL job to create all global vertex and edge types, and data sources.
+* `global.gsql` - DDL job to create all global vertex and edge types, data sources, packages, template queries and functions.
```

Comment:
@xun-gao-tigergraph @dadongwang-tg While there is nothing wrong with the techincal information, where there is an output or behavioral change, such as in this case, we should add a statement in the documentation, such as 
"In 4.1+, the contents of the Export package have been expanded to include packages, template queries, and used defined functions."